webui/react/src/pages/ModelVersionDetails/ModelVersionHeader.tsx

```diff
@@ -220,11 +220,8 @@ with det.import_from_path(path + "/code"):
         onCancel={() => setShowUseInNotebook(false)}>
         <div className={css.topLine}>
           <p>Reference this model in a notebook</p>
-          <ClipboardButton getContent={() => referenceText} />
         </div>
-        <pre className={css.codeSample}>
-          <code>{referenceText}</code>
-        </pre>
+        <CodeSample text={referenceText} />
         <p>Copy/paste code into a notebook cell</p>
```

Comment:
nit: since this `<p>` doesn't have the spacing applied, suggesting they be removed from the other modals too. might as well stay consistent and reduce context-level css at the same time. If vertical spacing does matter, that should probably be handled with a `Column` component in the future, anyway.

Reply:
✅